src/custom/messages.ts

```diff
@@ -0,0 +1,26 @@
+export class Messages {
+    static MODE = "basic";
+
+    get(messageIdentifier: string): string {
+        return msgConfig[messageIdentifier][Messages.MODE];
+    }
+
+    setMODE(personalityName: string): void {
+        Messages.MODE = personalityName;
+    }
+}
+
+const msgConfig: { [key: string]: any } = {
```

Comment:
As part of these structure, I would like to replace the `any` here with an interface for each personality. Something like the following should work

```
interface Personalities {
  basic: string;
  brandon: string;
}
```